### PR TITLE
Issue 31/ft text image banner

### PR DESCRIPTION
**Pull request by Beulah-Matt**
# Description

This pull request is part of the work to make it easier for developers to create a complete about us section. 
It is a reusable component for most of the social impact sections

To achieve this, we needed to:

```
Create a reusable component
Use input decorators to capture content
```

Fixes # (issue) -  Fixes # [([S4Y-400](https://www.w3.org/Provider/Style/dummy.html), [S4Y-100](https://www.w3.org/Provider/Style/dummy.html))](https://github.com/italanta/elewa-group/issues/31)

## Type of change

Please delete options that are not relevant.

- [yes ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
imgright
imagerightbanner

# How Has This Been Tested?
Checked for functionality on my browser and compared my final product to the given design

# Checklist:
- [ yes] My code follows the style guidelines of this project
- [ yes] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ yes] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by LemmyMwaura — COMMENTED** [review]
This issue was closed as completed. #178

File: libs/features/components/elewa-group-image-and-text-banner/src/lib/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+@Component({
+  selector: 'elewa-group-image-and-text-banner',
+  templateUrl: './elewa-group-image-and-text-banner.component.html',
+  styleUrls: ['./elewa-group-image-and-text-banner.component.scss'],
+})
+export class ElewaGroupImageAndTextBannerComponent {
+  @Input() imagePlacement: string
+  @Input() title: string
+  @Input() imageUrl: string
+  @Input() paragraphText: string
```
Comment (IanOdhiambo9) [inline libs/features/components/elewa-group-image-and-text-banner/src/lib/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.ts:12]: Initialise to empty string/default value i.e.

@Input() paragraphText: string = '';

**Review by IanOdhiambo9 — COMMENTED** [review]
This PR contains a package-lock.json file.

cannot be merged.

**Comment by IanOdhiambo9** [comment]
I'm closing this PR because it's the wrong lib, contains package-lock.json and issue has been closed.

**Pull request closed without merging**